crates/stores/src/rocksdb/mod.rs

```diff
@@ -209,11 +210,11 @@ impl Store for RocksDB {
                 pools::add(&batch, add.pools)?;
                 accounts::add(&batch, add.accounts)?;
 
+                accounts::reset(&batch, withdrawals)?;
```

Comment:
Any reason for this change?

Reply:
Yes! See: f4f4a78eea11dfcb6a05e4ed2086efc7a7b77cfa